libs/consul/config.jsonnet

```diff
@@ -0,0 +1,27 @@
+local config = import 'jsonnet/config.jsonnet';
+
+local versions = ['0.34.1'];
```

Comment:
In every other library, we don't put the patch number in the `output` value. For consistency, I think it'd be better to omit it